src/Microsoft.Azure.SignalR.Common/ClientInvocation/ClientResultsManager.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Buffers;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Diagnostics;
+using System.Collections.Generic;
+using System.Collections.Concurrent;
+using Microsoft.Azure.SignalR.Protocol;
+using Microsoft.AspNetCore.SignalR.Protocol;
+using Microsoft.AspNetCore.SignalR;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal class ClientResultsManager : IClientResultsManager, IInvocationBinder
+    {
+        private readonly ConcurrentDictionary<string, PendingInvocation> _pendingInvocations = new();
+        private readonly ConcurrentDictionary<string, List<string>> _serviceMappingMessages = new();
+        private readonly string _serverGUID = Guid.NewGuid().ToString();
```

Comment:
every instance of ClientResultManager results in a different _serverGUID, is that intentional?